onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -1364,7 +1404,7 @@ TEST(PoolTest, LpPool) {
 
   test.AddInput<float>("X", x_dims, x_vals);
   test.AddOutput<float>("Y", expected_dims, expected_vals);
-  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kCudaNHWCExecutionProvider});
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kCudaExecutionProvider, kCudaNHWCExecutionProvider});
```

Comment:
Is there a reason why previously running tests are now disabled on NHWC CUDA EP ? 

The PR description says - "Disabled all pooling tests which are not supported by the CUDA EP." Can you please expand on what support is being pulled away from NHWC CUDA EP ? The NHWC CUDA EP is the default CUDA EP that ships in release binaries and we should be careful of feature regression.

Reply:
Neither the CUDA EP nor the NHWC CUDAEP supports the LpPool operation. This is a feature which has to be added.

Reply:
Oh I see. I guess we don't need to explicitly disable it then as it will never run on the CUDA EP (because there is no kernel registered) and hence it can not "fail" ?

Reply:
This way it's quite easy to find all tests which are either unsupported or fail. I'd prefer to keep it this way.